packages/design-system/src/components/SkipNav/SkipNav.stories.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import React from 'react';
+
+import SkipNav from './SkipNav';
+
+export default {
+  title: 'Components/Skip Nav',
+  component: SkipNav,
+};
+
+const Template = ({ data, ...args }) => <SkipNav {...args} />;
+
+export const DefaultSkipNav = Template.bind({});
+
+export const SkipNavExample = Template.bind({});
+SkipNavExample.decorators = [
+  (Story) => (
+    <div>
+      {Story()}
+      <nav className="ds-u-padding--3">
+        <h1 className="ds-u-margin--0">Navigation</h1>
+        <p className="ds-u-margin--0">Clicking on the Skip Nav will skip over this section.</p>
+        <ul>
+          <li>
+            <a href="javascript:void(0);">Navigation link 1</a>
+          </li>
+          <li>
+            <a href="javascript:void(0);">Navigation link 2</a>
```

Comment:
I see a console error that reads
```
react-dom.development.js:88 Warning: A future version of React will block javascript: URLs as a security precaution. Use event handlers instead if you can. If you need to generate unsafe HTML try using dangerouslySetInnerHTML instead. React was passed "javascript:void(0);".
```
Consider updating the href of these to something like `#` or `google.com`

Reply:
Thanks for catching this - I went with "#".